homeassistant/components/recorder/models.py

```diff
@@ -256,16 +259,36 @@ class Statistics(Base):  # type: ignore
     sum = Column(DOUBLE_TYPE)
     sum_increase = Column(DOUBLE_TYPE)
 
-    @staticmethod
-    def from_stats(metadata_id: str, start: datetime, stats: StatisticData):
+    @classmethod
+    def from_stats(cls, metadata_id: str, start: datetime, stats: StatisticData):
         """Create object from a statistics."""
-        return Statistics(
+        return cls(  # type: ignore
```

Comment:
mypy doesn't like this, possibly because of the `metadata_id` which is no longer a `Column`.